stdlib/filepath/src/main/ballerina/src/filepath/file_path.bal

```diff
@@ -129,12 +147,15 @@ public function parent(string path) returns string|Error {
     return validatedPath.substring(0, len);
 }
 
-# Returns the shortest path name equivalent to path by purely lexical processing.
+# Returns the shortest path name equivalent to the path by purely lexical processing.
```

Comment:
This sounds odd. Shall we rephrase this?

Reply:
Corrected.